src/components/sidebar/sidebar.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import '../../styles.css';
+import './sidebar.css';
+import defaultImg from '../../assets/defaultImg.png';
+
+export default function Sidebar() {
+    return (
+        <div className="sidebar-container">
+            <div className="menu-options">
+                <div><i className="fa-solid fa-house"></i> Home</div>
```

Comment:
for icons u need not use `div`. `span` would be enough. keep it semantic 😃

Reply:
Done 👍🏼